src/scores/probability/crps_impl.py

```diff
@@ -219,35 +223,39 @@ def crps_cdf(
         fcst: array of forecast CDFs, with the threshold dimension given by `threshold_dim`.
         obs: array of observations, not in CDF form.
         threshold_dim: name of the dimension in `fcst` that indexes the thresholds.
-        threshold_weight: weight to be applied along `threshold_dim` to calculat
-            threshold-weighted CRPS. Must contain `threshold_dim` as a dimension, and may
-            also include other dimensions from `fcst`. If `weight=None`, a weight of 1
+        threshold_weight: weight to be applied along `threshold_dim` to calculate 
+            threshold-weighted CRPS. Must contain `threshold_dim` as a dimension, and may 
+            also include other dimensions from `fcst`. If `weight=None`, a weight of 1 
             is applied everywhere, which gives the usual CRPS.
-        additional_thresholds: additional thresholds values to add to `fcst` and (if
+        additional_thresholds: additional thresholds values to add to `fcst` and (if 
             applicable) `threshold_weight` prior to calculating CRPS.
-        propagate_nans: If `True`, propagate NaN values along `threshold_dim` in `fcst`
-            and `threshold_weight` prior to calculating CRPS. This will result in CRPS
-            being NaN for these cases. If `False`, NaN values in `fcst` and `weight` will
-            be replaced, wherever possible, with non-NaN values using the fill method
+        propagate_nans: If `True`, propagate NaN values along `threshold_dim` in `fcst` 
+            and `threshold_weight` prior to calculating CRPS. This will result in CRPS 
+            being NaN for these cases. If `False`, NaN values in `fcst` and `weight` will 
+            be replaced, wherever possible, with non-NaN values using the fill method 
             specified by `fcst_fill_method` and `threshold_weight_fill_method`.
-        fcst_fill_method: how to fill values in `fcst` when NaNs have been introduced
-            (by including additional thresholds) or are specified to be removed (by
-            setting `propagate_nans=False`). Select one of:
-                - "linear": use linear interpolation, then replace any leading or
-                  trailing NaNs using linear extrapolation. Afterwards, all values are
-                  clipped to the closed interval [0, 1].
-                - "step": apply forward filling, then replace any leading NaNs with 0.
-                - "forward": first apply forward filling, then remove any leading NaNs by
-                  back filling.
-                - "backward": first apply back filling, then remove any trailing NaNs by
-                  forward filling.
-            In most cases, "linear" is likely the appropriate choice.
+        fcst_fill_method: how to fill values in `fcst` when NaNs have been introduced 
```

Comment:
Not sure why there is a line break for this one (and after `threshold_weight_fill_method`) in the docs, but not others

Reply:
There's no clear logic, but it made the rendering better. Without it, there's an inconsistent-looking grey box around the bullet list. So this is a test-driven change...